py/strato/common/log/log2text.py

```diff
@@ -122,51 +178,29 @@ def _getNextParsableEntry(inputStream, logFile, colorCode, formatter):
     list the file until the next parsable line
     finish when all lines were listed
     """
+    logTypeConf = formatter._getLogTypeConf(logFile)
     while True:
         try:
             line = inputStream.next()
-            obj = json.loads(line)
-            formatted = formatter.process(obj)
-            return obj, None if formatted is None else _addLogName(formatted, colorCode, logFile)
+            formatted, timestamp = formatter.process(line, logFile, logTypeConf)
+            return None if formatted is None else _addLogName(formatted, colorCode, logFile), timestamp
```

Comment:
baaa.... now u return 2 values here? why?
Also, in case of an exception of StopIteration, you'll return single value.